deployment/ccip/changeset/cs_add_chain_test.go

```diff
@@ -138,7 +138,7 @@ func TestAddChainInbound(t *testing.T) {
 	}, []commonchangeset.ChangesetApplication{
 		// note this doesn't have proposals.
 		{
-			Changeset: commonchangeset.WrapChangeSet(commonchangeset.NewTransferOwnershipChangeset),
+			Changeset: commonchangeset.WrapChangeSet(commonchangeset.TransferToMCMSWithTimelock),
```

Comment:
Ditto here re: axing the next ChangesetApplication in this slice, since this already gens the accept ownership + executes